src/hooks/useUserGrants.ts

```diff
@@ -1,18 +1,18 @@
-import { useUser } from 'context/UserContext';
 import { getUserGrants } from 'api/userGrants';
 import { singleCallSettings } from 'context/SWR';
-import { useSelectNew } from './supabase-swr/hooks/useSelect';
+import { useQuery } from '@supabase-cache-helpers/postgrest-swr';
+import { useUserStore } from 'context/User/useUserContextStore';
 
 function useUserGrants(singleCall?: boolean) {
-    const { session } = useUser();
+    const user = useUserStore((state) => state.user);
```

Comment:
New way to fetch user as we can fetch that directly instead of pulling it from the session. This is nice because sometimes the session can change but the user stays the same.